### ISSUE TITLE
error tool/ module not found

### ISSUE THREAD

**Issue by tothuhien** [usegalaxy-no/galaxyadmin#82]
It seems that from late last friday, all jobs failed with different errors but in general is:
tool/module etc ... not found. 
While those tools used to work before. I have rerun some previously successful jobs and now they all failed. 
Is there update on the system lately?

**Comment by tothuhien** [comment]
As Kjetil pointed out that it failed to call the specific container of each tool, so always the default container was used - which obviously do not contain any package requirement for the tool.
The log file pointed out error was caused by the function `list_cached_mulled_images_from_path` of the file `/srv/galaxy/server/lib/galaxy/tool_util/deps/container_resolvers/mulled.py`. This function is supposed to list all the available containers in the folder `/srv/galaxy/containers/singularity/mulled` - in order to find the container name for the tool. It failed because there're some manually created containers with the name do not respect the name rule that they should be. For example: `mulled-v2-2dfb284e80d1e71bb82a6298b977e01b29ba3dc0.sif_kjetil`. I have now moved those containers to another place `/srv/galaxy/containers/singularity/test` and it seems to fix the problem.
